challenges/frontend/challenge.md

```diff
@@ -228,6 +228,12 @@ These are the areas we will be evaluating in the submission:
 To help you out, we’ve composed a list of things that previously resulted in a no-pass from the interview team:
 
 * Scope creep. Candidates have tried to implement too much and ran out of time.
+   * Avoid implementing an overly complex solution just to show that you are capable of writing a
+     complex feature. Instead, if you think something could be made more complex in a full-fledged
+     app, leave a comment about it and move on with a solution which solves the problem at hand.
+   * For example, there is no need to implement a pluggable auth system which in the future would
+     let you easily switch between different auth methods. Focus on implementing a single auth
+     method that satisfies challenge requirements.
```

Comment:
> Focus on implementing a single auth method that satisfies challenge requirements.

This sentence makes it sound as if I wanted to say "Aha, be careful! There are some auth methods which do not satisfy challenge requirements!". But I don't really know how I could say this without suggesting that there are some hidden gotchas.

Reply:
idk, maybe something like this?
> It is better to focus on implementing a single auth method.